.github/workflows/template_secret_scan.yml

```diff
@@ -0,0 +1,26 @@
+name: Secret Scanning
+
+on:
+  workflow_call:
+
+jobs:
+  trufflehog:
+
+    name: trufflehog
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: TruffleHog OSS
+        uses: trufflesecurity/trufflehog@v3.4.3
+        with:
+          # Repository path
```

Comment:
Could you remove these comments?

Reply:
Would you like to have removed all comments? We thought it would be helpful to leave the "head comment" at least as it is not self-explaining.